config/index.js

```diff
@@ -755,6 +755,12 @@ var conf = convict({
       default: false,
       format: Boolean,
       env: 'SECONDARY_EMAIL_ENABLED'
+    },
+    minUnverifiedAccountTime: {
+      doc: 'The minimum amount of time an account can be unverified before another account can add it\'s email',
```

Comment:
` before another account can add it\'s email',` -> ` before another account can use it for secondary email`,

could be `consume it` than `use it`?